docs/source/quickstart.rst

```diff
@@ -0,0 +1,114 @@
+Quickstart
+==========
+
+
+Install
+-------
```

Comment:
Should we actually move installation to its own section? Could be saved for a subsequent PR.

Reply:
done